internal/pipeline/resolver/picker.go

```diff
@@ -4,4 +4,11 @@ import (
 	"github.com/buildkite/cli/v3/internal/pipeline"
 )
 
+// PipelinePicker is a function used to pick a pipeline from a list.
+//
+// It is indended to be used from pipeline resolvers that resolve multiple pipelines.
 type PipelinePicker func([]pipeline.Pipeline) *pipeline.Pipeline
+
+func PassthruPicker(p []pipeline.Pipeline) *pipeline.Pipeline {
+	return &p[0]
+}
```

Comment:
This is the temporary fix I assume?

Reply:
Yeah and maybe just for tests as well. Intentionally basic (and not checking nil) just to show example usage. We probably should delete this once we have "real" pickers